assets/pipeline-slack-notification-lambda/index.py

```diff
@@ -64,6 +64,13 @@ class TriggerMetadata(t.TypedDict):
     vcs: TriggerMetadataVcs
 
 
+def mask_string(input_string, num_chars_to_mask=20, mask_character="*"):
```

Comment:
Clean implementation (although I think it should be accompanied by a docstring if we're going to use it 🖊️). Here's an alternative implementation that you may consider that makes us less prone to changes in the format on Slack's end (e.g., they increase the length of the final path segment to 30 characters).

```suggestion
def get_masked_slack_webhook_url(slack_webhook_url: str):
    """
    Return a string that masks the final path segment of a Slack webhook URL.

    The URL is typically formatted as such: https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX
    """
    trimmed_url = slack_webhook_url.rstrip("/")
    [*url, final_path_segment] = trimmed_url.split("/")
    return "/".join(url + [len(final_path_segment) * "*"])
```

Reply:
Bruker dette i stedet, lurt å ta hensyn til at Slack kan gjøre endringer på format